modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/poll/PollingSourceWrapper.java

```diff
@@ -754,6 +792,29 @@ private void shutdownScheduler() {
     }
   }
 
+  private int matchAction(PollContext.PollItemStatus status, boolean currentPollItemLimitApplied) {
```

Comment:
improve method name. matchAction to what? `statusToNotificationType` is probably a better name